app/src/organisms/Devices/ProtocolRun/SetupLabware/__tests__/SetupLabwareList.test.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
+import { when } from 'jest-when'
+import { fireEvent } from '@testing-library/react'
+import { i18n } from '../../../../../i18n'
+import _protocolWithMagTempTC from '@opentrons/shared-data/protocol/fixtures/6/transferSettings.json'
+import { renderWithProviders } from '@opentrons/components'
```

Comment:
not sure about the order of `_protocolWithMagTempTC` and `renderWithProviders` but i18n should be after them, right?

```suggestion
import { renderWithProviders } from '@opentrons/components'
import _protocolWithMagTempTC from '@opentrons/shared-data/protocol/fixtures/6/transferSettings.json'
import { i18n } from '../../../../../i18n'
```

Reply:
Yes, i agree with you. good catch. I am unsure where the `_protocolWithMagTempTC` should go but before i18n makes sense